lib/buffy/throttle_and_timed.ex

```diff
@@ -0,0 +1,372 @@
+# credo:disable-for-this-file Credo.Check.Refactor.LongQuoteBlocks
+defmodule Buffy.ThrottleAndTimed do
+  @moduledoc """
+  This is a variation on the `Buffy.Throttle` behavior.
+
+  It keeps the following functionality:
+    - wait for a specified amount of time before
+  invoking the work function. If the function is called again before the time has
+  elapsed, it's a no-op.
+
+  Key difference between `Buffy.Throttle` and `Buffy.ThrottleAndTimed`:
+    - it will not be terminated once the timer is done, but kept alive
+      - internally, the existing timer behavior is done via state rather than handling `{:error, {:already_started, pid}}` output of `GenServer.start_link`.
+        - See note on Horde about state.
+    - it requires `:loop_interval` field value (set by config) to trigger work repeatedly based on a empty inbox timeout interval,
+      that is based on [GenServer's timeout feature](https://hexdocs.pm/elixir/1.12/GenServer.html#module-timeouts).
```

Comment:
Bumping doc version in URL. If I didn't already have other suggestions, I wouldn't have added this 😆 🤷🏻 

```suggestion
      that is based on [GenServer's timeout feature](https://hexdocs.pm/elixir/1.15/GenServer.html#module-timeouts).
```